13/umbraco-cms/fundamentals/backoffice/login.md

```diff
@@ -56,7 +56,7 @@ The **Forgotten password?** link allows your backoffice users to reset their pas
 
 Set it to `true` to enable the password reset feature, and `false` to disable the feature.
 
-You will also need to configure an Simple Mail Transfer Protocol (SMTP) server in your `appsettings.json` file. When you get a successful result on the SMTP configuration when running a health check in the backoffice, you are good to go!
+You will also need to configure an outbound e-mail server (SMTP) in your `appsettings.json` file. When you get a successful result on the SMTP configuration when running a health check in the backoffice, you are good to go!
```

Comment:
@jonat123 Before it used both the long form of the technology "Simple Mail Transfer Protocol" and the acronym "SMTP", but that didn't work with Vale, nor does this work now without being allowed the "SMTP" acronym. Can you give me some tips to describe this in a better way, please?

Reply:
In this case I think you can just use the initial version with the Simple Mail Transfer Protocol (SMTP) definition.

As simple is a part of the acronym the warning in this case should just be ignored 😊

Reply:
Fixed, thanks!